packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -157,18 +150,35 @@ export const buildFilterInputType = (modelType: GraphQLObjectType) => {
   });
 }
 
-export const buildCreateMutationInputType = (modelType: GraphQLObjectType) => {
+// Fetching all fields required for input type including relationships
+const getAllModelFields = (modelType: GraphQLObjectType<any, any, { [key: string]: any; }>, model: ModelDefinition) => {
+  const modelFields = Object.values(modelType.getFields());
+  const scalarFields = modelFields.filter((f: GraphQLField<any, any>) => isScalarType(getNamedType(f.type)));
+  const relationshipFields = model.relationships.map((relationship: FieldRelationshipMetadata) => {
+    return {
+      name: getInputFieldName(relationship.ownerField),
```

Comment:
This is throwing an 'Not inputtable error' because you are trying to map a 1:N field to an input, but this is invalid (inputs should only accept scalar with reference to foreign key field). Basically this should be filtered out before this mapping is applied.

Reply:
Since this functions gets all model fields and not necessarily input fields, there should be no need to map the fields to input fields at this point.